rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -2632,7 +2632,7 @@ private HalResource<Project> createHalLicenseClearing(Project sw360Project, List
 		sw360.unsetVisbility();
 		sw360.unsetSecurityResponsibles();
 		HalResource<Project> halProject = new HalResource<>(sw360);
-
+        halProject.add(linkTo(ProjectController.class).slash("api/projects/" + sw360Project.getId()).withSelfRel());
```

Comment:
Please fix the indentation.

```suggestion
		halProject.add(linkTo(ProjectController.class).slash("api/projects/" + sw360Project.getId()).withSelfRel());
```